scripts/assemble_xc_framework.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/zsh
+
+cd ../
+./gradlew :app-io-shared:assembleSharedReleaseXCFramework
+./gradlew :app-io-shared:assembleSharedDebugXCFramework 
+echo -e "\a"
```

Comment:
I specified this because I want a sound to be played when the output is completed.